ctapipe/reco/stereo_combination.py

```diff
@@ -102,10 +105,10 @@ class StereoMeanCombiner(StereoCombiner):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        if self.property not in {"energy", "classification"}:
+        if self.property not in {"energy", "classification", "geometry"}:
```

Comment:
It would be better to use an Enum here rather than an ad-hoc string.  Here all possibly values are known in advance, so it's not arbitrary.  Enums are better for typos and speed (later you do a lot of compares per event to  e.g.  `if self.property == 'geometry':`

Same for the Reconstructor itself, where there is `property="geometry"`

Reply:
THough this is more general than this PR, and  seems that this will be addressed  in #2116 , so I think we can ignore it here.

Reply:
Yes, let's merge this in and implement #2116 in one go for all the reconstructors